core/tests/unit/ldml/keyboards/k_103_vkey.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+@@keys: [K_A][K_U][K_Q][K_C][K_K][K_SLASH][K_Z][K_Q][K_L][K_T][K_W]
```

Comment:
@mcdurdin  is this right?  AUQCK/ZQLTW -> quack/waltz

maybe this is partly a test question. do we expect process_key to 'honor' the vkey mappings at this level?

Reply:
Yes, I think these test key events should be based on the US ANSI vk codes.